src/app/ngrx/effects/routing.effects.ts

```diff
@@ -36,9 +36,6 @@ export class RoutingEffects {
         if (params['endDate']) {
           filter.endDate = new Date(params['endDate']);
         }
-        if (params['range']) {
-          filter.range = params['range'].split(',');
-        }
```

Comment:
added a few more files to the PR but I went ahead and removed this `range` support that was for the `prev` and `next` buttons since those are now gone with these layout changes as well

Reply:
+1